lego/apps/events/tests/test_events_api.py

```
@@ -509,6 +517,15 @@ class CreateEventsTestCase(BaseAPITestCase):
         "test_users.yaml",
         "test_events.yaml",
     ]
+    PAYLOAD_FIELDS = [
+        "title",
+        "description",
+        "text",
+        "event_type",
+        "location",
+        "start_time",
+        "end_time",
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)